src/main/java/com/sprint/mission/discodeit/repository/BinaryContentRepository.java

```diff
@@ -0,0 +1,27 @@
+package com.sprint.mission.discodeit.repository;
+
+import com.sprint.mission.discodeit.entity.BinaryContent;
+import jakarta.annotation.PostConstruct;
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.PersistenceContext;
+import jakarta.transaction.Transactional;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface BinaryContentRepository extends JpaRepository<BinaryContent,UUID> {
+
+  BinaryContent save(BinaryContent binaryContent);
+
+  Optional<BinaryContent> findById(UUID id);
+
+  List<BinaryContent> findAllByIdIn(List<UUID> ids);
+
+  boolean existsById(UUID id);
+
+  void deleteById(UUID id);
```

Comment:
JpaRepository를 상속받으면 JpaRepository에서 기본적으로 제공하는 메서드는 모두 제거해도 될 것 같아요.  
```suggestion
  List<BinaryContent> findAllByIdIn(List<UUID> ids);
```

`List<BinaryContent> findAllByIdIn(List<UUID> ids);`는 CrudRepository 내 `Iterable<T> findAllById(Iterable<ID> ids);`로 대체 가능합니다!